packages/image_picker/image_picker_ios/example/ios/RunnerTests/PickerSaveImageToPathOperationTests.m

```diff
@@ -40,7 +40,7 @@ - (void)testSaveJPGImage API_AVAILABLE(ios(14)) {
                                                              withExtension:@"jpg"];
   NSItemProvider *itemProvider = [[NSItemProvider alloc] initWithContentsOfURL:imageURL];
   PHPickerResult *result = [self createPickerResultWithProvider:itemProvider
-                                                 withIdentifier:UTTypeWebP.identifier];
+                                                 withIdentifier:UTTypeJPEG.identifier];
```

Comment:
Does the `itemProvider.registeredTypeIdentifiers` first item match these identifiers? Would be better to not have to specify it and for `createPickerResultWithProvider` to figure it out from the provider.

Reply:
Good idea. I'll try that